File: submissions/favouroked/factorial.js
var number;
var factorial;
function find_factorial(x){
  if (number > 1){
    x *= number;
    number -= 1;
    return find_factorial(x);
  } else {
    return x;
  }
}

function factorial(value, func){
  number = value-1;
  fact = func(value);
  return fact;
}

console.log(factorial(150, find_factorial));

File: submissions/TaslimOseni/Factorial.java
public class Factorial {

	
	//Factorial method
	int factorialMethod(int number){
		if((number == 1) || (number == 1)){
			return 1;
		}
		return number * factorialMethod(number - 1);
	}
	

	
	//Loop method
	int loopMethod(int number){
		if((number == 1) || (number == 1)){
			return 1;
		}
		int factorial = 1;
		for(int i = 1; i <= number; i++){
			factorial *= i;
		}
		return factorial;
	}

}

